composer.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "moxie-leean/wp-elements",
```

Comment:
@mitogh 
All instances except this one or we wont be able to get it to packagist, right?